internal/driver/macmapper_test.go

```diff
@@ -178,3 +182,232 @@ func TestMatchEndpointRefAddressToMAC(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Remove Underscore in name and add a comment to the test case above to be consistent with other unit tests.